tests/openvino/test_modeling.py

```diff
@@ -528,6 +528,7 @@ class OVModelForCausalLMIntegrationTest(unittest.TestCase):
         "mistral",
         "mixtral",
         "mpt",
+        # "olmo", # TODO : enable after optimum update for support transformers 4.40
```

Comment:
```suggestion
        "olmo",
```
patch release is out : https://pypi.org/project/optimum/#history

Reply:
@eaidova let's enable this test before merging